### PR TITLE
alternate for find -E on non-macos

### PR DESCRIPTION
**Pull request by pedritomelenas**
find -E is not suitable for ubuntu

### CODE REVIEW

**Comment by fingolfin** [comment]
Thanks! But I am afraid that doesn't quite cut it, as some people also use GNU find on macOS; and potentially there are *BSD users out there... we could test if it is GNU find or BSD find..... but the real fix will be to rewrite the regex with the "old style" regex syntax *sigh*

File: release-gap-package [hunk 1]
```diff
@@ -452,7 +452,11 @@ rm -rf .git* .hg* .cvs* .circleci
 rm -f .mailmap
 rm -f .appveyor.yml .codecov.yml .travis.yml azure-pipelines.yml
 # remove TeX leftovers
-find -E doc -regex '.*\.(aux|bbl|blg|brf|idx|ilg|ind|log|out|pnr|toc|tst)' -exec rm -f {} +
+if [ "$(uname)" == "Darwin" ]; then
+  find -E doc -regex '.*\.(aux|bbl|blg|brf|idx|ilg|ind|log|out|pnr|toc|tst)' -exec rm -f {} +
```
Comment (fingolfin) [inline release-gap-package:456]: Probably should do `find doc -name '*.aux' -o -name '*.bbl' -o ...`

Comment (pedritomelenas) [reply release-gap-package:456]: `find doc -name '*.aux' -o -name '*.bbl' -o -name '*.blg' -o -name '*.brf' -o -name '*.idx' -o -name '*.ilg' -o -name '*.ind' -o -name '*.log' -o -name '*.out' -o -name '*.pnr' -o -name '*.toc' -o -name '*.tst'` will do

**Comment by fingolfin** [comment]
Thanks!

**Pull request merged**